controllers/logicalvolume_controller.go

```diff
@@ -51,6 +52,8 @@ func (r *LogicalVolumeReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 	if lv.ObjectMeta.DeletionTimestamp == nil {
 		// When lv.Status.Code is not codes.OK (== 0), CreateLV has already failed.
 		// LogicalVolume CRD will be deleted soon by the controller.
+		// In the case of retry of other operations than CreateLV,
+		// the controller should clear lv.Status.Code.
```

Comment:
Why do you implement so?
Think simply, if the controller can retry the resizing operation, the node controller can retry it by itself, doesn't it?